### ISSUE TITLE
Next version of puf.csv file

### ISSUE THREAD

**Issue by martinholmer** [open-source-economics/taxdata#96]
@MattHJensen wants to split up the activities mentioned in issue #95 so that a new version of the **SAS-generated** `puf.csv` file is available no later than **this Thursday**.  So, the things that need to get done in the next day or so include the following:

- remove *ad hoc* age-difference logic from `puf_data/finalprep.py` in anticipation of the new python-generated `cps-matched-puf.csv file that contains more sensible ages for taxpayer and spouse (@martinholmer in PR #yy)

- add nu18 variable (@hdoupe in PR #93)

- replace net casualty losses with gross casualty losses (@martinholmer in PR #94)

In addition, PR #93 and PR #94 will require changes in Tax-Calculator, which I will be happy to handle.  In fact, the Tax-Calculator code changes to handle taxdata #94 have already been prepared in [taxcalc pull request 1426](https://github.com/open-source-economics/Tax-Calculator/pull/1426).

@andersonfrailey, What version of the raw SAS-generated `cps-matched-puf.csv file are we currently using?  On my computer I have this `cps-matched-puf.csv file:
```
puf_data$ ls -l cps-m*
-rw-r--r--@ 1 mrh  staff  178345362 Mar 21 13:33 cps-matched-puf.csv

puf_data$ awk -F, 'NR>1{n++}END{print n}' cps-matched-puf.csv 
219806
```
It has eight fewer records than the current version of the `puf.csv` file.
Is that the file that the stages and finalprep work will begin with this week?
I want to make sure I have the same raw starting file as everybody else.

@MattHJensen @Amy-Xu @andersonfrailey @hdoupe

**Comment by andersonfrailey** [comment]
From #92 in response to @martinholmer:

>@martinholmer, the raw cps-matched-puf.csv file you're looking at (with 219,806 records) was created with the SAS scripts. We never established why running the scripts on my machine versus John's resulted in eight fewer records that the current puf.csv (there was a previous issue discussing this, but I haven't found it yet. I'll post a link when I do).
The raw file created from the python scripts are creating a PUF with 215,525 records and that's what we will be starting with.

**Comment by martinholmer** [comment]
taxdata issue #96 is being resolved by pull request #100.
